e2e/vertical-full-page-map.spec.ts

```diff
@@ -49,20 +62,32 @@ test.describe('full page map test suite', () => {
   });
 
   test('default initial search works and is enabled by default', async ({ page }) => {
-    await page.getByPlaceholder('Search for locations').press('Enter');
-    const result = page.locator('#js-answersVerticalResults div').nth(0);
-    await expect(result).toBeAttached();
+    const response = await page.waitForResponse(resp =>
+      resp.url().includes('https:\/\/prod-cdn\.us\.yextapis\.com\/v2\/accounts\/me\/search\/vertical\/query')
+      && resp.url().includes('queryTrigger'));
+    expect(response.status()).toBe(200);
   });
 
+  test('empty search works', async ({ page }) => {
+    await page.getByPlaceholder('Search for locations').press('Enter');
+    const response = await page.waitForResponse(resp =>
+      resp.url().includes('https:\/\/prod-cdn\.us\.yextapis\.com\/v2\/accounts\/me\/search\/vertical\/query')
+      && resp.url().includes('input=&'));
+    expect(response.status()).toBe(200);
+  })
+
   test('pagination works', async ({ page }) => {
+    const firstPage = page.locator('#js-answersVerticalResultsCount');
+    await expect(firstPage).toHaveText(/20/);
     await page.getByLabel('Go to the next page of results').click();
     const secondPage = page.locator('#js-answersVerticalResultsCount');
     await expect(secondPage).toHaveText(/21/);
   });
 
   test('pagination scrolls the results to the top', async ({ page }) => {
-    await page.getByLabel('Go to the next page of results').click();
     const topOfResults = page.locator('#js-answersVerticalResults div').nth(0);
+    await expect(topOfResults).not.toBeVisible();
```

Comment:
out of curiosity, how come the top of the results isn't visible when we haven't scrolled down the page before this?

Reply:
on second thought in response to your previous comment about checking if the top of page is not visible before we go to the next page of results, i don't think that would be reasonable in this test until we added in actions for manually scrolling to the bottom of the page, since with playwright we can just use a locator to find the pagination button. i think it just makes sense for this test to check if the first result card is visible after clicking the button and not if the first result isn't visible before clicking it.

Reply:
based on the name of this test, I thought the purpose was to check that the scrolling offset resets itself after pagination? if you're already at the top of the page and then click the pagination button, it won't actually test that the scroll offset has reset to 0, because it was already 0 to begin with. we likely would need an action for manually scrolling the page first